### PR TITLE
Feature: reintroduce iframe view

### PR DESCRIPTION
**Pull request by riedde**
While scanning the code I found some fragments of an iframe view. This PR is a contribution to reintroduce it.

### CODE REVIEW

**Comment by bwbohl** [comment]
@riedde, what’s the status of this draft?
@Edirom/edirom-online-developer is there a need for an iFrameView – showing external resources might be a useful feature.

**Comment by riedde** [comment]
I'm not sure about my intention I had on this PR. Maybe to remodel a feature that has been lost at some day. Maybe I also tried to view the content of another eXist-App in an Edirom window (as the RWA did years ago).

**Comment by DennisFriedl** [comment]
Might be usefull in some cases. Was it removed in the past?

**Comment by riedde** [comment]
Maybe deactivated or lost somehow. But not fully deleted, fragments are still there. So I think it wasn't planned to cut it out.

**Comment by nikobeer** [comment]
It should still work, i think. We still use it. For example here, when you click on „Textvergleich“ in the navigator: https://www.reger-werkausgabe.de/edition.html?workID=rwa_work_00036. It will open an object window containing an iFrame that is loading our text comparison tool.

Side note: Edirom-Online within RWA Online is provided within an iFrame. So: RWA Online -> iFrame Edirom-Online -> iFrame text comparison.

File: add/data/xql/getLinkTarget.xql [hunk 1]
```diff
@@ -142,7 +146,9 @@ declare function local:getWindowTitle($doc as node()+, $type as xs:string) as xs
   (: Work :)
   if(exists($doc//mei:mei) and exists($doc//mei:workDesc/mei:work) and not(exists($doc//mei:perfMedium)))
   then(eutil:getLocalizedTitle(($doc//mei:work)[1]/mei:titleStmt[1], $lang))
-
+  else if(exists($doc/root()/mei:work))
```
Comment (roewenstrunk) [inline add/data/xql/getLinkTarget.xql:149]: This addition has nothing to do with the iFrame view. Could you please remove this change from this PR, @riedde?

File: add/data/xql/getLinkTarget.xql [hunk 2]
```diff
@@ -142,7 +146,9 @@ declare function local:getWindowTitle($doc as node()+, $type as xs:string) as xs
...
+  then(eutil:getLocalizedTitle($doc/root()/mei:work, $lang))
```
Comment (roewenstrunk) [inline add/data/xql/getLinkTarget.xql:150]: Same here

**Comment by riedde** [comment]
@roewenstrunk Do we have progress on this? If yes, could you push the changes we have done at our meeting?

**Review by roewenstrunk — APPROVED** [review]

**Pull request merged**